### PR TITLE
Remove use of styled-components in desktop/quicklook

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
None

**Description**
Remove use of styled-components

### CODE REVIEW

File: desktop/quicklook/GlobalStyle.tsx [hunk 1]
```diff
@@ -2,42 +2,48 @@
 // License, v2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import { createGlobalStyle } from "styled-components";
+import { Global, css } from "@emotion/react";
```
Comment (jtbandes) [inline desktop/quicklook/GlobalStyle.tsx:5]: Why not use tss-react for this?

File: desktop/quicklook/FileInfoDisplay.tsx [hunk 2]
```diff
@@ -30,133 +32,112 @@ function formatTimeRaw(stamp: Time): string {
...
+    display: "inline-block",
+    width: 40,
+  },
+  topicList: {
+    wordBreak: "break-word",
+    borderSpacing: "0 4px",
+
+    [`@media (max-width: ${NARROW_MAX_WIDTH}px)`]: {
+      marginLeft: -10,
+      marginRight: -10,
```
Comment (jtbandes) [inline desktop/quicklook/FileInfoDisplay.tsx:74]: Please don't delete the `styleConstants.bodyPadding` here. It was used in multiple files and now is a magic number that will be impossible to remember to change in all relevant places in the future.

File: desktop/quicklook/FileInfoDisplay.tsx [hunk 3]
```diff
@@ -30,133 +32,112 @@ function formatTimeRaw(stamp: Time): string {
...
-    display: none;
-  }
-`;
-const ShowNarrow = styled.div`
-  @media (min-width: ${styleConstants.breakpoints.narrowMinWidth}) {
-    display: none;
-  }
-`;
-
-function TopicRow({ info: { topic, datatype, numMessages, numConnections } }: { info: TopicInfo }) {
```
Comment (jtbandes) [inline desktop/quicklook/FileInfoDisplay.tsx:146]: Still seems preferable to have this broken into a separate component for readability. The FileInfoDisplay component is pretty gigantic.

**Review by jtbandes — APPROVED** [review]

File: desktop/quicklook/FileInfoDisplay.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import { useEffect, useMemo } from "react";
-import styled from "styled-components";
+import { makeStyles } from "tss-react/mui";
```
[the remark this answers is not in the document]
Comment (2metres) [reply desktop/quicklook/FileInfoDisplay.tsx:6]: I can't seem to make it happy importing it from anywhere else

**Pull request merged**